bin/ansible-connection

```diff
@@ -89,10 +89,9 @@ class ConnectionProcess(object):
         self._ansible_playbook_pid = ansible_playbook_pid
 
     def start(self, variables):
+        messages = list()
```

Comment:
Is there a particular reason for these changes? I don't really see what moving these into the `try:` block does for anyone